packages/widgets/src/health-monitoring/component.tsx

```diff
@@ -0,0 +1,319 @@
+                align="center"
+                key={integrationId}
+              >
+                <Box className="health-card-info">
+                  <HoverCard>
+                    <HoverCard.Target>
+                      <Indicator
+                        inline
+                        processing
+                        color={healthInfo.rebootRequired ? "red" : healthInfo.availablePkgUpdates > 0 ? "blue" : "gray"}
+                        position="top-end"
+                        size="4cqmin"
+                        label={healthInfo.availablePkgUpdates > 0 ? healthInfo.availablePkgUpdates : undefined}
+                        disabled={!healthInfo.rebootRequired && healthInfo.availablePkgUpdates === 0}
+                      >
+                        <Avatar size="10cqmin" radius="sm">
+                          <IconInfoSquare size="10cqmin" />
+                        </Avatar>
+                      </Indicator>
+                    </HoverCard.Target>
+                    <HoverCard.Dropdown>
+                      <List size="sm" center spacing="0.5cqmin">
+                        <List.Item icon={<IconCpu2 size="1cqmin" />}>
+                          {t("widget.healthMonitoring.popover.processor")}: {healthInfo.cpuModelName}
+                        </List.Item>
+                        <List.Item icon={<IconVersions size="1cqmin" />}>
+                          {t("widget.healthMonitoring.popover.version")}: {healthInfo.version}
+                        </List.Item>
+                        <List.Item icon={<IconClock size="1cqmin" />}>
+                          {t("widget.healthMonitoring.popover.uptime")}: {formatUptime(healthInfo.uptime)}
+                        </List.Item>
+                      </List>
+                    </HoverCard.Dropdown>
+                  </HoverCard>
+                </Box>
+                {options.cpu && (
+                  <Box ref={ref} w="100%" h="100%">
+                    <RingProgress
+                      className="health-card-cpu"
+                      roundCaps
+                      size={ringSize}
+                      thickness={ringThickness}
+                      label={
+                        <Center style={{ flexDirection: "column" }}>
+                          <Text size="3cqmin">{`${healthInfo.cpuUtilization.toFixed(2)}%`}</Text>
+                          <HoverCard>
+                            <HoverCard.Target>
+                              <IconCpu size="7cqmin" />
+                            </HoverCard.Target>
+                            <HoverCard.Dropdown>
+                              <List size="sm" center spacing="0.5cqmin" icon={<IconCpu2 size="1cqmin" />}>
+                                <List.Item icon>{t("widget.healthMonitoring.popover.loadAverage")}</List.Item>
+                                <List.Item>
+                                  1 {t("widget.healthMonitoring.popover.minute")}: {healthInfo.loadAverage["1min"]}
```

Comment:
You should put the number and the ``:`` also in a translation.
In Arabic and other languages they will be reversed - hence they should be in the translations